src/web/mjs/connectors/NovelCool.mjs

```diff
@@ -0,0 +1,75 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Mangafest extends Connector {
```

Comment:
Classname does not match meta data

Reply:
May create a template similar to WordPressMadara instead and make implementations for each support language